tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,43 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _wait_element = (By.CLASS_NAME, 'collections')
```

Comment:
Ideally we'd wait on a displayed element, or even something that a user could interact with. For example we could wait for `len(self.collections) > 0` or for `self.collections[0].is_displayed`.